cmd/info.go

```diff
@@ -3,12 +3,10 @@ package cmd
 import (
 	"fmt"
 
-	"github.com/Benchkram/errz"
 	"github.com/puppetlabs/wash/api/client"
 	cmdutil "github.com/puppetlabs/wash/cmd/util"
 	"github.com/puppetlabs/wash/config"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 )
 
 func infoCommand() *cobra.Command {
```

Comment:
Could we alias this to `winfo` ?

Reply:
Hmm, in my case it's not builtin, it's just at /usr/bin/info. Which shell are you using?

Reply:
Oh, I meant mine is also at `/usr/bin/info`. I'm using Bash.

If `info` isn't used that often, OK to keep it as-is

Reply:
It seems to be an alternative to `man`. I've never used it.

Reply:
May as well alias it then. `winfo` is still pretty short

Reply:
The same argument can be made for `clear`. Should I change that back to `wclear` as well, since it doesn't do the same thing?

Reply:
Guess that makes sense. Done.

Reply:
Yeah, definitely for `clear`. I use clear a lot in my shell.